tests/evergreen/test_task.py

```diff
@@ -8,6 +8,12 @@
 
 
 class TestTask(object):
+    def test_evg_task_has_no_estimated_cost(self, sample_task):
+        assert not sample_task.get("estimated_cost")
+        task = Task(sample_task, None)
+        with pytest.raises(AttributeError):
+            assert task.estimated_cost
+
```

Comment:
This test generates a `sample_task` which does not have `estimated_cost` field. This should raise an `AttributeError` if that field is accessed on the `Task` object.